drivers/jdbc-driver/src/main/kotlin/com/squareup/sqldelight/sqlite/driver/JdbcDriver.kt

```diff
@@ -68,6 +69,16 @@ abstract class JdbcDriver : SqlDriver {
       transactions.set(enclosingTransaction)
     }
   }
+
+  companion object {
+    operator fun invoke(dataSource: DataSource): JdbcDriver {
```

Comment:
Also, in general, not a fan of `operator invoke` outside of DSLs. The discoverability is shit.

What about a top-level
```kotlin
@file:JvmName("JdbcDrivers")

@JvmName("fromDataSource")
fun DataSource.asJdbcDriver() = ...

class JdbcDriver { .. }
```
instead?